filebeat/prospector/log/file.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-type LogFile struct {
+type File struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type File should have comment or be unexported